LNK4217: Warning when building vsprojects

We see tons of these warnings, it pollutes the test output.
```
libeay32.lib(c_zlib.obj) : warning LNK4217: locally defined symbol deflate imported in function BIO_f_zlib [c:\jenkins\workspace\gRPC_portability_master\language\c\scenario\windows_x64_vs2013\vsprojects\vcxproj\test\h2_oauth2_bad_hostname_test\h2_oauth2_bad_hostname_test.vcxproj
```

--- a/templates/vsprojects/global.props.template
+++ b/templates/vsprojects/global.props.template
@@ -13,6 +13,10 @@
         <PreprocessorDefinitions>_WIN32_WINNT=0x600;_SCL_SECURE_NO_WARNINGS;_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
         <WarningLevel>EnableAllWarnings</WarningLevel>
       </ClCompile>
+      <Link>
+        <!-- LNK4271 pollutes test output. See #4521 -->
+        <AdditionalOptions>/ignore:4217 %(AdditionalOptions)</AdditionalOptions>
+      </Link>
     </ItemDefinitionGroup>
     <ItemGroup />
   </Project>

--- a/vsprojects/global.props
+++ b/vsprojects/global.props
@@ -11,6 +11,10 @@
       <PreprocessorDefinitions>_WIN32_WINNT=0x600;_SCL_SECURE_NO_WARNINGS;_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>EnableAllWarnings</WarningLevel>
     </ClCompile>
+    <Link>
+      <!-- LNK4271 pollutes test output. See #4521 -->
+      <AdditionalOptions>/ignore:4217 %(AdditionalOptions)</AdditionalOptions>
+    </Link>
   </ItemDefinitionGroup>
   <ItemGroup />
 </Project>

